### DISCUSSION TITLE
Remove GLMakie.Screen() requirement for opening a new window.

### DISCUSSION THREAD

**Discussion by nathanrboyer** [JuliaPlots/Makie.jl#4249] [Ideas]
This was suggested in the original pull request but not implemented for some reason.
https://github.com/MakieOrg/Makie.jl/pull/1771#issuecomment-1145918227

Will you consider having `display(fig1); display(fig2)` open two windows without the need to remember/write `display(GLMakie.Screen(), figure_or_scene)`? It seems to me that a figure should always be associated with a separate window.

If not automatically every time, then maybe something easier to remember like a keyword argument `display(fig2, new_window=true)`?

(I would like to see the AlgebraOfGraphics `draw` function automatically open new windows too. I don't know if that would happen implicitly with this change or not.)

**Comment by jkrumbiegel** [comment]
This seems like something that should be a settable preference to me. Some people will prefer a new window every time, some won't because it clutters their workspace. I also don't think `display(GLMakie.Screen(), figure_or_scene)` is easy to remember.

**Comment by dav2017** [comment]
You can also create your own shortcut command, eg.:

    dispnew(figure) = display(GLMakie.Screen(), figure)

**Comment by bc0n** [comment]
I was glad to discover just now that Makie can display multiple windows, which is to say that the single-window default behavior is not what I expected. For instance, Matlab uses the figure handle to control where data is plotted, drawing the window at the first figure() or plot() command without an explicit display().  
If `display()` has different arguments it should clearly open multiple windows, while calling display multiple times on the same figure is redundant or an indirect attempt to animate the window.
